libtiff/bitarray-a1646c0/bitarray/test_bitarray.py

```diff
@@ -124,7 +130,7 @@ def getIndicesEx(r, length):
 
 # ---------------------------------------------------------------------------
 
-class TestsModuleFunctions(unittest.TestCase, Util):
+class TestsModuleFunctions(Util):
```

Comment:
Because `Util` was made a child of ``unittest.TestCase`` to enable ``self.assertRaises`` to work, and presence of both `Util` and ``unittest.TestCase`` cause problems with method resolution order. [This](https://stackoverflow.com/questions/29214888/typeerror-cannot-create-a-consistent-method-resolution-order-mro) is where I learned of a fix.